lib/src/widgets/slider_title.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+class SliderTitle extends StatelessWidget {
+  const SliderTitle(
+    this.title,
+    this.text, {
+    Key? key,
+  }) : super(key: key);
+
+  final String title;
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 34.0,
+      child: Row(
+        children: <Widget>[
+          Opacity(
+            opacity: 0.7,
+            child: Align(
+              alignment: Alignment.bottomCenter,
+              child: Padding(
+                padding: const EdgeInsets.only(left: 10.0),
```

Comment:
It will be better if we support RTL.

Reply:
I aligned the right padding to 10.0 to make it symmetric.

Reply:
Hmm, then a `Padding` or `Container` would be better than a `SizedBox`.

Reply:
You got me thinking and I refactored the widget, now it should be better with RTL layout.